### DISCUSSION TITLE
Cilium with Singularity

### DISCUSSION THREAD

**Discussion by anylevel** [sylabs/singularity#2417] [Q&A]
Is it possible to use Cilium (without an orchestration system) with Singularity. Are there any installation instructions?

**Comment by dtrudg** [comment]
Hello. The issue has been moved to a Q&A Discussion as it is a usage question, rather than bug report or detailed feature request.

SingularityCE supports CNI based networking in setuid mode. See https://docs.sylabs.io/guides/4.0/user-guide/networking.html#network

It may be possible to do what you wish. The Cilium network would have to be setup separately, and you would need to write a CNI configuration file appropriate for a Singularity container to join it.

I'm afraid there is no documentation for this, and I'm not aware of anyone who has configured Singularity with Cilium.
